Makefile

```diff
@@ -33,7 +33,7 @@ envinit:
 # Initializiation on MacOS
```

Comment:
It seems we have a problem.

Above on line 31 there is:

```
# The gomobile/gobind libs are also needed when using `gomobile bind`, but it's not compatible with vendoring:
	# https://github.com/golang/go/issues/50994#issuecomment-1032754206
	GO111MODULE=off go get -u golang.org/x/mobile/cmd/gomobile
```

This fails in Go 1.22 because `go get` is now disabled outside of modules. I locally built an image simply removing this line, but then `cd backend/mobileserver; make` fails with

```
unable to import bind: no Go package in golang.org/x/mobile/bind
"golang.org/x/mobile/bind" is not found; run go get golang.org/x/mobile/bind: no Go package in golang.org/x/mobile/bind
```

Running `go get golang.org/x/mobile/bind` did not actually fix it for me.

Any idea how to resolve this?

Reply:
I also tried adding a dummy import `_ "golang.org/x/mobile/bind"` to force it to be vendored, but `go mod vendor` only vendors files in Go folders, but `bind` has files like these that are then missing:

```
gomobile: /opt/go/bin/gobind -lang=go,java -outdir=/tmp/gomobile-work-2993124781 github.com/BitBoxSwiss/bitbox-wallet-app/backend/mobileserver failed: exit status 1
failed to open Java support file: open /opt/go/src/github.com/BitBoxSwiss/bitbox-wallet-app/vendor/golang.org/x/mobile/bind/java/Seq.java: no such file or directory

```

I filed an issue here: https://github.com/golang/go/issues/67927

Reply:
A workaround that seems to work is to `mv vendor vendor.disabled` before `make android` and move it back after :scream_cat: